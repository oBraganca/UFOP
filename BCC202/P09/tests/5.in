5
MariaHelena 575774 7.6
Joaquim 870575 2.7
AnaJulia 825049 1.5
PedroHenrique 51326 8.0
Caleb 825294 6.2
