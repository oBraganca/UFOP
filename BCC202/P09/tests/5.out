AnaJulia 825049 1.5
Joaquim 870575 2.7
Caleb 825294 6.2
MariaHelena 575774 7.6
PedroHenrique 51326 8.0
